### ISSUE TITLE
accessibility: toggle button contrast

### ISSUE THREAD

**Issue by derekeder** [chihacknight/il-solar-map#47]
From https://github.com/chihacknight/il-solar-map/pull/45#issuecomment-2118302996

The toggle button `active` does not have enough contrast for good accessibility.

**Comment by derekeder** [comment]
this is due to the custom colors in the Zephyr Boostrap skin: https://bootswatch.com/zephyr/
